### DISCUSSION TITLE
Mapa do Evento

### DISCUSSION THREAD

**Discussion by MathRMM** [diego3g/rsxp-2023#123] [Q&A]
Estou criando uma feat trazendo um mapa da expo norte, com a ideia de mostrar onde vão estar localizados as atividades nos dias dos eventos. Já tem uma Issue em aberto na qual detalho bem melhor a ideia e como estou fazendo, mas estou precisando de ajuda para o design já que quase nenhuma experiência nessa área. As minhas dúvidas ficam na base de qual cor usar e como definir as coisas na parte interna do mapa, se alguém puder me dar uma força vou ficar muito grato.
#107

**Comment by Rodrigo001-dev** [comment]
Fala @MathRMM, Blz?
Cara eu não sou design e por isso não vou poder opindar muito mas por que tu não utiliza aquelas cores que tu tinha colocado na issue?

No caso essa aqui
![Screenshot from 2023-04-29 04-45-40](https://user-images.githubusercontent.com/59906558/235291667-a3734b27-7138-4bfa-a1da-2de141a0c267.png)

Mas poderiamos adaptar essas cores para as cores da Rocket, como no caso o objetivo é somente mostrar as atividades do dia, ai poderiamos listar todas as atividades que vai ter no evento e definir qual cor vai ser utilizada para cada tipo de atividade, o que tu acha?
